6.5_group_by.py

```diff
@@ -0,0 +1,20 @@
+def group_by(f, iterable):
+    """
+    get a function and an iterable and returns a dictionary whose keys are
+     the activation of the function on each member and the values are the members
+    :param f: function for key
+    :param iterable: iterable
+    :return: group dictionary by function
+    """
+    group_dictionary = {}
+    for i in iterable:
+        if f(i) in group_dictionary:
+            group_dictionary[f(i)].append(i)
```

Comment:
Replace 'i' and 'f' Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing

Reply:
I changed i to item and f to func